static/app/routes.tsx

```diff
@@ -274,6 +274,11 @@ function buildRoutes() {
         component={make(() => import('sentry/stories/view/index'))}
         withOrgPath
       />
+      <Route
+        path="/notifs/"
+        component={make(() => import('sentry/debug/notifs/views/index'))}
```

Comment:
opted to nest `notifs` in `debug` (even though its the only member) to make it explicit nothing here should make it into the main app, this is only internal tooling.